Wrappers/Python/cil/optimisation/functions/KullbackLeibler.py

```diff
@@ -372,7 +372,7 @@ def kl_div_mask(x, y, eta, mask):
                     accumulator[get_thread_id()] += Y
                 else:
                     # out.flat[i] = numpy.inf
-                    return numpy.inf
+                    accumulator[get_thread_id()] = numpy.inf
```

Comment:
```suggestion
                    accumulator[get_thread_id()] = numpy.inf
                    break
```

Reply:
`break` creates a multiple exit scenario. Using `pass` instead would work, but it doesn't change. 

I see an option of creating an array for checking whether for each thread we calculated `inf` once and then avoid the calculation.